uavcan/tunnel/Protocol.uavcan

```diff
@@ -4,5 +4,8 @@
 #
 
 uint8 MAVLINK                   = 0     # MAVLink
+uint8 MAVLINK2                  = 1     # MAVLink2
+uint8 GPS                       = 2     # GPS
```

Comment:
Wouldn't the protocol be NMEA or UBX or something like that?

Reply:
Ardupilot autodetects the specific protocol. I guess a better definition would be `GPS_GENERIC`